### ISSUE TITLE
Additional newline after tall inventory output

### ISSUE THREAD

**Issue by c64-adventures** [johanberntsson/PunyInform#111]
Just updated to the latest build, and It looks like an extra newline is being added after the player's inventory list, when displayed using tall `inventory_style`. Everything still looks fine when using the wide style.

**Comment by fredrikr** [comment]
It looks correct to me. Image shows testbench.inf.

Can you show an example of how it goes wrong?

As we have reworked PrintContents a lot, it's not at all unlikely there are issues regarding inventory.

![image](https://user-images.githubusercontent.com/1001548/188285648-b71e5a3e-c7ab-4b13-b87c-43c8ad80d0ad.png)

**Comment by c64-adventures** [comment]
That makes sense now I've gone back to look at my code.

I've got a LibraryMessages routine that's overriding the default MSG_INVENTORY_DEFAULT behavior. In [this recent commit](https://github.com/johanberntsson/PunyInform/commit/eba795cf21c8df6f13b6e03e4b2373a84b3d1e06) messages.h has been updated so that the newline is only displayed if `inventory_stage` is wide, whereas before it would show in both cases.

I've now reflected that change in my LibraryMessages code and it's displaying fine again in tall mode.

Thanks!
